README.md

```diff
@@ -114,22 +114,21 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://deploy-preview-31--rentbikes.netlify.app)
+- [Live Demo Link]
```

Comment:
- [ ] I understand this is a work in progress.  I really recommend you correct the frontend tests, or provide instructions on your readme about how to correctly run them, in case there are special procedures to be taken in order to make them pass. 👍 

![image](https://user-images.githubusercontent.com/57421823/230703929-51a0b957-953e-4a01-a99d-20b557ff6918.png)